NewSemVer.ps1

```diff
@@ -24,9 +24,8 @@ $semVerPattern = '\d+.\d+.\d+(-[a-zA-Z0-9-]+)?'
 Print usage info.
 #>
 function Usage {
-    "Updates the AssemblyVersion, AssemblyInformationalVersion and AssemblyFileVersion in "
-    "the SharedVersionInfo.cs file. Then, tries to create a new NuGet package, which will fail if "
-    "nuget.exe isn't in the PATH or the script's directory.`n"
+    "Updates the Version in the .csproj file. Then, tries to create a new NuGet package, which will"
+    "fail if nuget.exe isn't in the PATH or the script's directory.`n"
```

Comment:
This comment is wrong, since we aren't directly using `nuget.exe` but rather rely on the dotnet core tools being installed, so I'll need to fix that.

Reply:
Fixed in another commit.